src/App.tsx

```diff
@@ -21,7 +30,9 @@ const authConfig: AuthConfiguration = {
 function App() {
   return (
     <RootProviders authConfig={authConfig}>
-      <RootStack />
+      <PushNotificationScreenProvider>
+        <RootStack />
+      </PushNotificationScreenProvider>
```

Comment:
As mentioned above, I don't think this is the correct implementation.

If the end goal here is to have someone copy this repo and have a notifications "playground", I think you have to create a new navigation stack to render your `PushNotificationsScreen` alongside the RootStack. But from a consumer standpoint, this doesn't really make sense. You would never view notifications without being signed in.